astropy/coordinates/sky_coordinate.py

```diff
@@ -1792,9 +1792,9 @@ def radial_velocity_correction(
 
           >>> vcorr_rel = vcorr.to(u.Hz, u.doppler_optical(1*u.Hz)).to(vcorr.unit, u.doppler_relativistic(1*u.Hz))  # doctest: +REMOTE_DATA
 
-        See also `~astropy.units.equivalencies.doppler_optical`,
-        `~astropy.units.equivalencies.doppler_radio`, and
-        `~astropy.units.equivalencies.doppler_relativistic` for more information on
+        See also `~astropy.units.doppler_optical`,
```

Comment:
It's good to update these, regardless! We had private API leaking into our public API. Anyone looking at the docs could copy-paste this mistake into their code.